### PR TITLE
Remove obsolete code

### LINKED ISSUE #118: Remove obsolete code
After #106 goes in, a large chunk of code can be removed:

* [x] mesh example
* [x] the `project` module
* [x] everything related to GPG

Note that this will also break the CLI. As it is for demo purposes only, I suggest to remove it entirely for now, and resurrect it as needs arise and time permits.

### PR DESCRIPTION
**Pull request by kim**
Closes #118

### CODE REVIEW

File: librad/src/git/repo.rs [hunk 1]
```diff
@@ -422,6 +426,14 @@ impl Repo {
         })
     }
 
+    /// Obtain a [radicle-surf] `Browser` for this repo, consuming self.
+    ///
+    /// [radicle-surf]: https://github.com/radicle-dev/radicle-surf
+    pub fn browser(self) -> Result<surf::Browser, Error> {
+        let bro = surf::Browser::new(self.repo.into())?;
```
Comment (FintanH) [inline librad/src/git/repo.rs:433]: `bro` and `Ok(bro)` - amazing :clap: :smile:

File: librad/src/keys.rs [hunk 2]
```diff
@@ -15,5 +15,274 @@
...
+
+    pub fn public(&self) -> PublicKey {
+        PublicKey(self.0.public_key())
+    }
+
+    pub fn sign(&self, data: &[u8]) -> Signature {
+        Signature(ed25519::sign_detached(data, &self.0))
+    }
+
+    const PKCS_ED25519_OID: &'static [u64] = &[1, 3, 101, 112];
```
Comment (FintanH) [inline librad/src/keys.rs:71]: What's this? :thinking:

File: librad/src/keys.rs [hunk 3]
```diff
@@ -15,5 +15,274 @@
...
+    }
+}
+
+impl AsRef<[u8]> for SecretKey {
+    fn as_ref(&self) -> &[u8] {
+        self.0.as_ref()
+    }
+}
+
+#[derive(Debug)]
```
Comment (FintanH) [inline librad/src/keys.rs:114]: Would it be useful to have an `Error` instance?

File: librad/src/keys.rs [hunk 4]
```diff
@@ -15,5 +15,274 @@
...
+        let bytes = hex::decode(s)?;
+        let buffer = if bytes.len() == 64 {
+            let mut buffer = [0u8; 64];
+            for (i, v) in bytes.iter().enumerate() {
+                buffer[i] = *v;
+            }
+            buffer
+        } else {
+            return Err(signature::DecodeError::InvalidLength {
+                actual: bytes.len(),
```
Comment (FintanH) [inline librad/src/keys.rs:231]: minor suggestion: Have a `let actual = bytes.len();` above to reuse the result in the `if` and here :)

File: librad/src/keys.rs [hunk 5]
```diff
@@ -15,5 +15,274 @@
...
+    }
+
+    pub fn from_bs58(s: &str) -> Option<Self> {
+        let bytes = match bs58::decode(s.as_bytes())
+            .with_alphabet(bs58::alphabet::BITCOIN)
+            .into_vec()
+        {
+            Ok(v) => v,
+            Err(_) => return None,
+        };
```
Comment (FintanH) [inline librad/src/keys.rs:244]: The indentation threw me off here. I think you could do:
```
let bytes = bs58::decode(s.as_bytes())
     .with_alphabet(bs58::alphabet::BITCOIN)
     .into_vec()
     .ok()?;
```

File: .buildkite/docker/rust/Dockerfile [hunk 6]
```diff
@@ -1,25 +1,46 @@
...
+    apt-get autoremove; \
+    rm -rf /var/lib/apt/lists/*
 
-RUN rustup component add rustfmt \
-    && cargo install sccache cargo-deb \
-    && rm -rf /usr/local/cargo/registry \
-    && rm /usr/local/cargo/.package-cache
+# Rust toolchain
+# Make sure this is in sync with rust-toolchain!
+ENV RUST_VERSION=nightly-2020-05-11 \
```
[the remark this answers is not in the document]
Comment (kim) [reply .buildkite/docker/rust/Dockerfile:23]: Ya maybe, but I think `COPY` doesn't follow symlinks, and can't escape the "context dir" (ie. this directory).

I'm contemplating a top-level [multistage](https://docs.docker.com/develop/develop-images/multistage-build/) `Dockerfile`, but need to figure out how to teach CI to only use the first stage.

File: librad/src/keys.rs (hunk 2 above)
Comment (kim) [reply librad/src/keys.rs:71]: It's a magical value encoding the signature algorithm in a PKCS-encoded certificate. I cannot document this, I'm sorry, it is only accessible to initiated members of the free masons guild "Templars of Londres Shacklewell" (TLS).

File: librad/src/keys.rs (hunk 5 above)
Comment (kim) [reply librad/src/keys.rs:235]: I did not write this. But it should also be obsolete soon (#134 + #121 + #115)

File: librad/src/keys.rs (hunk 4 above)
Comment (kim) [reply librad/src/keys.rs:222]: This method is obsolete

**Pull request merged**